presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -2,6 +2,38 @@
 Deploying Presto
 ================
 
+.. _requirements:
+
+Requirements
+------------
+
+.. _requirements-linux:
+
+Linux Operating System
+^^^^^^^^^^^^^^^^^^^^^^
+
+* 64-bit required
+* newer release preferred, especially when running on containers
+
+.. _requirements-java:
+
+Java Runtime Environment
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+* 64-bit required
+* version 11 recommended
+* higher versions receive limited testing
+* Azul Zulu used for builds and testing
+* AdoptOpenJDK, Amazon Corretto, Azul Zulu, and Oracle Java are known to be used
```

Comment:
Let's remove this. It's implying they work, which we don't know for certain, especially for any arbitrary version of Presto. We'd also need to include the RedHat version (the version installed with RHEL / CentOS)

Reply:
ok

Reply:
Maybe add Red Hat OpenJDK ?

Reply:
I'd rather just remove these since we don't test on them. The answer for Corretto is that it might work, but we don't test on it, so try it at your own risk.

The Presto Docker image uses the Red Hat one today, but we should switch it to Zulu.

Reply:
I also don't know anyone using Oracle Java as it is now commercial.